corehq/ex-submodules/casexml/apps/phone/restore.py

```diff
@@ -510,7 +518,7 @@ def get_cached_payload(self):
             return
 
         if self.sync_log:
-            payload = self.sync_log.get_cached_payload(self.version)
+            payload = self.sync_log.get_cached_payload(self.version, stream=True)
```

Comment:
This returns a File like object which streams the cached attachment contents from CouchDB.